cmd/configure.go

```diff
@@ -8,56 +8,113 @@ import (
 	"github.com/spf13/viper"
 )
 
-var configColors []*string
-
-var defaultColorKeys []string
-
-var defaultColors = map[string]string{
-	"gray":  "8",
-	"red":   "1",
-	"green": "2",
+// colorCfg holds config data for a single color
+// the `configValue` field is meant to be populated during the `init()` call to
+// be used by flags
+type colorCfg struct {
+	colorName    string
+	defaultValue string
+	configValue  *string
```

Comment:
can `configValue` just be a string and dereferenced in init?

```suggestion
type color struct {
	name         string
	defaultVal string
	configVal  string
```

Reply:
I'll check. I tried something similar but didn't work (but probably that was before I made it into a struct).